EasyInventory/src/main/resources/templates/start.html

```diff
@@ -1,10 +1,18 @@
 <!DOCTYPE html>
```

Comment:
For this you could add Bootstrap CSS and JS. And then make HTML a bit better by putting the EasyInventory in h1 tag

Reply:
Bootstrap is a good idea, but I would not add the script import until you implement Bootstrap in the page.  Otherwise, you have unused code, which is clutter, and that increases technical debt.